src/script/api/script_road.hpp

```diff
@@ -71,6 +84,19 @@ class ScriptRoad : public ScriptObject {
 		BT_TRUCK_STOP, ///< Build a truck stop
 	};
 
+	/**
+	 * Get the name of a road sub type.
+	 * @param road_type The road type to get the name of.
+	 * @param road_sub_type The road sub type to get the name of.
+	 * @pre IsRoadTypeAvailable(road_type, road_sub_type).
+	 * @return The name the road type has.
+	 * @note Since there is no string with only the name of the road, the text which
+	 *  is shown in the dropdown where you can chose a road type is returned. This
+	 *  means that the name could be something like "Road construction" instead
+	 *  of just "Road".
```

Comment:
this seems like a deficiency... can this not be solved?

Reply:
Yeah, seems odd to not include it

Reply:
I think it's wrong, too, there is a string with just the name.

Reply:
Yeah, it was added later. I wonder if it's mandatory. Hmm.